src/app/static/src/scss/style.scss

```diff
@@ -102,38 +69,3 @@ a {
 .custom-file, .custom-file label {
   overflow: hidden;
 }
-
-.sap-filters {
-  padding-top: 2px;
-}
-
-.vue-treeselect__multi-value-item {
-  background-color: $theme-red;
-  color: white;
-}
-
-.vue-treeselect__multi-value-item, .vue-treeselect__multi-value-item:hover {
-  .vue-treeselect__value-remove {
-    color: white;
-  }
-}
-
-.vue-treeselect__label-container, .vue-treeselect__label-container:hover {
-  .vue-treeselect__checkbox--checked, .vue-treeselect__checkbox--indeterminate {
-    border-color: $theme-red !important;
-    background: $theme-red !important;
-    width: auto;
-  }
-}
-
-.vue-treeselect {
```

Comment:
Mostly just moved these into the maps.scss partial, but removed this z-index rule and also the rule below - I couldn't see what the second one was doing but perhaps this was in error? What was the overflow-x auto rule doing? https://github.com/mrc-ide/hint/pull/48/files#diff-5a8656bcbf5cc2b00d87f69b9918fcc1L134